### PR TITLE
Generator: Update SDK /services/observability

### PR DESCRIPTION
**Pull request by stackit-pipeline**
Generated from GitHub run [10488693764](https://github.com/stackitcloud/stackit-api-specifications/actions/runs/10488693764)

### CODE REVIEW

File: services/observability/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
+## v0.12.0 (2024-08-21)
+- **Breaking change**: STACKIT Argus service has been renamed to STACKIT Observability
```
Comment (joaopalet) [inline services/observability/CHANGELOG.md:2]: I would start versioning from `v0.1.0` as this is a new service (similar to https://github.com/stackitcloud/stackit-sdk-go/blob/main/services/authorization/CHANGELOG.md)

Comment (joaopalet) [reply services/observability/CHANGELOG.md:2]: We could e.g. link the `argus` changelogs in case the user wants to look at them

Comment (joaopalet) [inline services/observability/CHANGELOG.md:2]: Also I think this not a breaking change, it's a release of a new module (the old `argus` module will still be usable with all its released versions/tags)

Comment (joaopalet) [reply services/observability/CHANGELOG.md:2]: In the Release Notes of the next GitHub release we could add something like a

> [!Warning]
> The STACKIT Argus service was renamed to Observability. There is a new `observability` service in the SDK which offers the same functionality as the deprecated `argus` service.
> SDK updates from now on will be released on the new `observability` service, meaning `argus` will no longer get updates.
> Please migrate to ...

Comment (GokceGK) [reply services/observability/CHANGELOG.md:2]: I changed the changelog and also added the highlight about renaming to the main changelog

Comment (GokceGK) [reply services/observability/CHANGELOG.md]: Changed version and linked the old argus changelogs like the authorization example

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,4 +1,17 @@
...
+> **The STACKIT Argus service was renamed to STACKIT Observability.** 
+> 
+> In the SDK, this means that there is a new `observability` service, which offers the same functionality as the deprecated `argus` service.
+> 
+> SDK updates from now on will be released on the new `observability` service, meaning `argus` will no longer get updates.
+> 
+> Please migrate to `github.com/stackitcloud/stackit-sdk-go/services/observability`.
+
+- `observability`: [v0.1.0](services/observability/CHANGELOG.md#v010-2024-08-21)
+  - First release. This module offers the same functionalities as `argus` (release [v0.11.0](https://github.com/stackitcloud/stackit-sdk-go/releases/tag/services%2Fargus%2Fv0.11.0)).
```
Comment (joaopalet) [inline CHANGELOG.md:14]: I would link to the changelog file here

Comment (joaopalet) [reply CHANGELOG.md:14]:
```suggestion
  - First release. This module offers the same functionalities as `argus` (release [v0.11.0](https://github.com/stackitcloud/stackit-sdk-go/blob/main/services/argus/CHANGELOG.md#v0110-2024-05-23).
```

File: services/observability/CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+## v0.1.0 (2024-08-21)
+First release.
+
+This module offers the same functionalities as `argus` (release [v0.11.0](https://github.com/stackitcloud/stackit-sdk-go/releases/tag/services%2Fargus%2Fv0.11.0)).
```
Comment (joaopalet) [inline services/observability/CHANGELOG.md:4]: Same here

```suggestion
This module offers the same functionalities as `argus` (release [v0.11.0](https://github.com/stackitcloud/stackit-sdk-go/blob/main/services/argus/CHANGELOG.md#v0110-2024-05-23).
```

**Review by joaopalet — APPROVED** [review]

**Pull request merged**